### PR TITLE
initialize projects

### PR DESCRIPTION
**Pull request by Abel-desu**
## to-do-list 

**This is a project that done by using**

- webpack 
- javascript html& css

### CODE REVIEW

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi @Abel-desu,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that does not have a descriptive pull request. You should add title and summary describing what you did in this pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  remove(id) {
+    this.toDoList = this.toDoList.filter((list) => list.id !== Number(id));
+  }
+}
+// Create new tasks in the todolist
+const lists = new Tasks();
+const todoDiv = document.querySelector('.lists');
+const getTodoList = () => {
+  todolist.forEach((list) => {
```
Comment (brytebee) [inline src/index.js:20]: - [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider this code: `Array.sort((a, b) => a.index - b.index`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Abel-desu👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Webpack was used to set up the project.✔️
- It has an array of simple objects containing the required keys and data types.✔️
- It renders the list to the page excellently.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
Hi @Abel-desu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**